lighthouse-core/lib/network-recorder.js

```diff
@@ -82,8 +82,14 @@ class NetworkRecorder {
   }
 
   onResourceChangedPriority(data) {
-    this.networkManager._dispatcher.resourceChangedPriority(data.requestId,
-      data.newPriority, data.timestamp);
+    // TODO: this.networkManager._dispatcher.resourceChangedPriority is set to
+    // undefined when onResourceChangedPriority is triggered in
+    // gatherers/offline.js. The underlying cause may need to be investigated.
+    // In the meantime, explicitly check that it's a function.
+    if (typeof this.networkManager._dispatcher.resourceChangedPriority === 'function') {
```

Comment:
i can't see simple a reason for this. 

resourceChangedPriority was introduced fairly recently: https://chromium.googlesource.com/chromium/src/+/ac43939a619f1ce77c43d0dba0fb379ff16013b1 (chromium rev 392154)

If it were possible to have a slightly older copy of devtools-frontend, then it would not have this function.
However our package.json requires `1.0.401423` which matches against chromium rev 401423 which is > 392154.

So to me there feels like two possibilities. @jeffposnick has an older frontend rev installed somehow accidentally. Or somehow else some magic is replacing the dispatcher with a slightly older version.